sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py

```diff
@@ -84,6 +84,7 @@
     python_requires=">=3.8",
     install_requires=[
         "azure-core<2.0.0,>=1.28.0",
+        "azure-identity~=1.17", #TODO: 1.0.0 works. 1.17 is latest that would not require azure core bump
```

Comment:
Is there a reason why we are not including this functionality in the distro rather than the exporter? Most of our configurations in terms of features are in the distro (like live metrics).

Reply:
The way autoinstrumentation grabs exporters, this is the most OTel-appropriate design. We could maybe do something weird the distro globalizes a credential, and the exporter just looks for it. But I think that would be really confusing.

Reply:
Also, just remembered, I left this TODO there to discuss when you got back. We have a few options for where to set the min version of a new dependency.